sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -110,10 +112,33 @@ class EventData(object):
 
     """
 
-    def __init__(self, body=None):
-        # type: (Union[str, bytes, List[AnyStr]]) -> None
+    @overload
+    def __init__(self, *, data: bytes, content_type: str, **kwargs):
+        ...
+
+    @overload
+    def __init__(self, body: Optional[Union[str, bytes, List[AnyStr]]] = None) -> None:
+        ...
+
+    def __init__(
+        self,
+        body: Optional[Union[str, bytes, List[AnyStr]]] = None,
+        *,
+        data: bytes = None,
+        content_type: str = None,
```

Comment:
@yunhaoling - Should we document that we take 'data' and 'content_type' as kwargs now? Do we want to keep this "private" for use in the encoder, since users don't really need to know about this?

Reply:
OOB: add to docstring + changelog